app/scripts/modules/amazon/src/loadBalancer/configure/classic/createClassicLoadBalancer.controller.ts

```diff
@@ -149,11 +150,16 @@ export class CreateClassicLoadBalancerCtrl extends CreateAmazonLoadBalancerCtrl
     }
     loadBalancer.healthCheck = healthCheck;
   }
+
+  public validateTimeout(value: number): boolean {
+    return value < this.loadBalancerCommand.healthInterval;
+  }
 }
 
 export const AWS_CREATE_CLASSIC_LOAD_BALANCER_CTRL = 'spinnaker.amazon.loadBalancer.classic.create.controller';
 module(AWS_CREATE_CLASSIC_LOAD_BALANCER_CTRL, [
   require('@uirouter/angularjs').default,
+  'ui.validate',
```

Comment:
Is there a reason we couldn't follow the existing pattern of just having the require here?